### PR TITLE
Update README.md 7X. 8.X changes

### PR DESCRIPTION
**Pull request by daminux**
Updated datatable release
move 7.X bad link and add 8.x based on CHANGELOG, COMPOSER.JSON review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ It is generally recommended to use the latest point release of the relevant bran
 
 | Elastica branch                                       | ElasticSearch | elasticsearch-php | PHP            |
 | ----------------------------------------------------- | ------------- | ----------------- | -------------- |
-| [7.x](https://github.com/ruflin/Elastica/tree/master) | 7.x           | ^7.0              | ^7.2 \|\| ^8.0 |
+| [8.x](https://github.com/ruflin/Elastica/tree/master) | 8.x           | ^8.4              | >=8.0 <8.2
```
Comment (sidz) [inline README.md]: The right link to 8.x branch is https://github.com/ruflin/Elastica/tree/8.x

Comment (daminux) [reply README.md]: @sidz Thank you for your feedback, I made this change based on the history of changes in the README during which I noted that the last version was systematically positioned on the master branch which itself is an alias which points to the branch of the latest version. I took this to be a convention. I misunderstand ?

Comment (ruflin) [reply README.md]: There was a recent change around 8.x where we renamed `master` to `8.x`. There is no master branch anymore.

Comment (daminux) [reply README.md]: @ruflin @sidz amended

**Review by ruflin — APPROVED** [review]
LGTM. Did you leave it in draft on purpose?

**Comment by ruflin** [comment]
Thank you!

**Pull request merged**